HotSpot/View Controllers/AddCarViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  AddCarViewController.m
+//  HotSpot
+//
+//  Created by aaronm17 on 7/23/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "AddCarViewController.h"
+#import "Parse/Parse.h"
+#import "Car.h"
+
+@interface AddCarViewController ()
+
+@property (weak, nonatomic) IBOutlet UIButton *defaultButton;
+@property (weak, nonatomic) IBOutlet UIImageView *carImage;
+@property (weak, nonatomic) IBOutlet UITextField *licensePlate;
+@property (weak, nonatomic) IBOutlet UITextField *carColor;
+
+@end
+
+@implementation AddCarViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    [self.defaultButton setSelected:NO];
+
+}
+
+- (void)addCar {
+    BOOL *selected = (BOOL *) [self.defaultButton isSelected];
+    Car *car = [Car new];
```

Comment:
can you create a custom initializer for Car that takes in an arguments all the fields contained in the car object? this way, you can initialize a new car with the image, license plate, etc., and would not have to set these properties individually.